erts/emulator/beam/jit/arm/instr_bs.cpp

```diff
@@ -56,18 +54,39 @@ int BeamModuleAssembler::emit_bs_get_field_size(const ArgSource &Size,
         return -1;
     } else {
         auto size_reg = load_source(Size, TMP2);
+        bool can_fail = true;
+
+        if (always_small(Size)) {
+            auto [min, max] = getClampedRange(Size);
+            can_fail = !(0 <= min && (max >> 52) == 0);
```

Comment:
Is there a macro or constant we can use instead of 52 here?

Reply:
I have replaced `52` with `(SMALL_BITS - ERL_UNIT_BITS)`, where `ERL_UNIT_BITS` is a new define I've added to `erl_bits.h`.